admin/build-macos-external-list.sh

```diff
@@ -93,13 +93,20 @@ if [ ! "X$EXESHARED" = "X" ]; then
 fi
 cat << EOF
 
+# Place the licenses for runtime dependencies
 install (DIRECTORY
 	../../admin/Licenses
 	DESTINATION share
```

Comment:
Does this change looks better?
```suggestion
	DESTINATION \${GMT_DATADIR}
```

Reply:
see below